### PR TITLE
[FLOC-4053] Increase CloudFormation stack node timeout

### PR DESCRIPTION
**Pull request by myechuri**
This PR addresses 3 issues reported by Rob after trying CloudFormation:

- Failed stack creation due to CloudFormation timing out waiting for Control Node to come up (with Flocker and Swarm running) in 600 seconds. The PR unifies timeouts in the template, now set to 900 seconds across the board. Tested deployment 5 times with the new template.

- CloudFormation Inputs page does not list AWS AccessKey and SecretAccessKey right after one another. This is caused because all Inputs are sorted alphabetically by name by CloudFormation. The PR attempts to rename inputs such that AccessKey and SecretAccessKey appear next to each other.

- CloudFormation does not mention that VolumeHub Input is optional. The PR attempts to call out VolumeHub as optional, and provides a default value in the template. 

- In addition to above three changes, added documentation so that future maintainers are aware of stack startup order.

### CODE REVIEW

**Comment by wallrj** [comment]
Thanks @myechuri 

The acceptance test will need updating to use the new input parameter names.

```
AWS_DEFAULT_REGION=us-west-1 VOLUMEHUB_TOKEN='...' \
KEY_PAIR='...' \
SECRET_ACCESS_KEY='...' \
ACCESS_KEY_ID='...' \
CLOUDFORMATION_TEMPLATE_URL='https://s3.amazonaws.com/installer.downloads.clusterhq.com/flocker-cluster.cloudformation.json' trial flocker.acceptance.endtoend.test_installer
flocker.acceptance.endtoend.test_installer
  DockerComposeTests
    test_docker_compose_up_postgres ... 
A client error (ValidationError) occurred when calling the CreateStack operation: Parameters: [AmazonSecretAccessKey, AmazonAccessKeyID, EC2KeyPair] must have values

```

The docs for running that test are here:
 * http://doc-dev.clusterhq.com/gettinginvolved/acceptance-testing.html#cloudformation-installer-tests

I guess we should create and prioritze an issue to run that test from Jenkins....nightly for now?

Please resubmit when you've got the test passing.

**Comment by myechuri** [comment]
Thanks, @wallrj ! Reproduced the test failure. Will update PR once test is fixed.

**Comment by myechuri** [comment]
```
ubuntu@ip-172-31-25-23:~/flocker$ KEEP_STACK=1 CLOUDFORMATION_TEMPLATE_URL=https://s3.amazonaws.com/installer.downloads.clusterhq.com/flocker-cluster.cloudformation.json KEY_PAIR=$KEY ACCESS_KEY_ID=$ACCESS_KEY_ID SECRET_ACCESS_KEY=$SECRET_ACCESS_KEY VOLUMEHUB_TOKEN='' trial flocker.acceptance.endtoend.test_installer
flocker.acceptance.endtoend.test_installer
  DockerComposeTests
    test_docker_compose_up_postgres ...
...
                                   [OK]

-------------------------------------------------------------------------------
Ran 1 tests in 619.382s

PASSED (successes=1)
ubuntu@ip-172-31-25-23:~/flocker$ 
```

**Comment by myechuri** [comment]
2 / 2 test runs passed. Second pass:

```
ubuntu@ip-172-31-25-23:~/flocker$ KEEP_STACK=1 CLOUDFORMATION_TEMPLATE_URL=https://s3.amazonaws.com/installer.downloads.clusterhq.com/flocker-cluster.cloudformation.json KEY_PAIR=$KEY ACCESS_KEY_ID=$ACCESS_KEY_ID SECRET_ACCESS_KEY=$SECRET_ACCESS_KEY VOLUMEHUB_TOKEN='' trial flocker.acceptance.endtoend.test_installer
flocker.acceptance.endtoend.test_installer
  DockerComposeTests
    test_docker_compose_up_postgres ... Warning: Permanently added '54.153.25.186' (ECDSA) to the list of known hosts.
                                   [OK]

-------------------------------------------------------------------------------
Ran 1 tests in 612.373s

PASSED (successes=1)
ubuntu@ip-172-31-25-23:~/flocker$ 
```

File: flocker/acceptance/endtoend/test_installer.py [hunk 1]
```diff
@@ -129,19 +129,21 @@ def wait_for_stack_status(stack_id, target_status):
...
+            target_status=target_status,
+            current_status=current_status
+        )
         if current_status == target_status:
             return stack_report
 
-    return loop_until(reactor, predicate, repeat(10, 60))
+    return loop_until(reactor,
+                      predicate,
+                      repeat(10, 120))
```
Comment (wallrj) [inline flocker/acceptance/endtoend/test_installer.py]: :question: This means wait for 20min, right? I guess we could just let this loop forever and let the 30 minute test timeout do its thing if the stack creation + tests take longer. We can decide on that when we see how long Jenkins takes to run the tests.

**Comment by wallrj** [comment]
Thanks @myechuri 

The acceptance test fixes and longer timeouts look good to me.

Please merge when you get that ebs driver builder to pass.

**Comment by myechuri** [comment]
Thanks for the quick review, @wallrj ! Started ``merge_pr``.

**Pull request merged**